redbox/models/settings.py

```diff
@@ -42,7 +42,6 @@ class Settings(BaseSettings):
 
     redis_host: str = "redis"
     redis_port: int = 6379
-    redis_password: str = "guest"
```

Comment:
i have removed this until we work out how to get auth working

Reply:
Is this mentioned somewhere on JIRA as something to fix?